distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1310,12 +1405,67 @@ private String ingestXmlWithProduct(String fileName) throws IOException {
         return metacardId;
     }
 
-    private void cleanProductCache() throws IOException {
-        FileUtils.cleanDirectory(Paths.get(ddfHome).resolve(PRODUCT_CACHE).toFile());
+
+    private Action getCswRetrievalHeaders(String filename) {
+        return composite(header("X-Csw-Product", "true"), header("Content-Disposition", "filename=" + filename));
+    }
+
+    private void verifyJson(JsonPath json, String attributePath, String expectedValue) {
```

Comment:
Should we put those CometD validation methods in a separate class under the `ddf.common.test.cometd` package instead of here?